### PR TITLE
feat : added deletenote functionality

### PR DESCRIPTION
**Pull request by GeetaPogul**
added a delete note functionality in Notes box

### CODE REVIEW

File: src/css/login.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.star{
+    color: var(--danger-color);
+}
+
+.btns{
+   border-style: none;
+   padding: 0 1.9rem;
+   height: 2.3rem;
+   margin: 0 2rem;
+   border-radius: .2rem;
```
Comment (aatishpandey) [inline src/css/login.css:47]: can use px here

File: src/css/main.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous");
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
```
Comment (aatishpandey) [inline src/css/main.css:3]: you have already imported in index.html, if not any specific use case , there is no need to import again

File: src/css/note.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  padding: 0.5rem 0.9rem;
+  position: relative;
+  text-align: end;
+}
+.new-note {
+  width: 17rem;
+  max-width: 17rem;
+  padding: 1rem;
+
+  padding-top: 0.3rem;
```
Comment (aatishpandey) [inline src/css/note.css:39]: same here, can use px for less than 10px

File: src/pages/Nav.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      {/* <Routes>
+        <Route path="/home" element={<Home />} />
+        <Route path="/docs" element={<Docs />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/label" element={<Label />} />
+        <Route path="/notes" element={<Notes />} />
+        <Route path="/trash" element={<Trash />} />
+        <Route path="/archive" element={<Archive />} />
+        <Route path="/signup" element={<Signup />} />
+      </Routes> */}
```
Comment (aatishpandey) [inline src/pages/Nav.js:62]: delete unnecessary comments

**Review by sashaboi — APPROVED** [review]
Looks good to merge! hoping to have netlify link

**Review by aatishpandey — APPROVED** [review]
Everything looks good.
Left some suggestions. Have a look

**Pull request merged**